Codeinwp/otter-blocks
# Add Modal Block

Closes https://github.com/Codeinwp/otter-internals/issues/175


### Summary


The same features like Popup except for Popup Settings tabs in the Inspector

#### Dynamic Overriding 

The `save` function of the modal saves only the inner content. The complete HTML structure is done via PHP, which overrides the default saving data. 

This method ensures that the modal is working under a valid license while keeping the original data safe. 

### Screenshots 

https://github.com/user-attachments/assets/23ff20c7-f2df-4a55-b9f3-47d7288f063e

----

### Test instructions


> [!NOTE]
> This is PRO block, you will need Otter Pro

Modal is a popup with only the anchor option.

1. Insert a button and set an anchor
2. Insert a modal with the anche for the button
3. Check if it works like in the video.



---- 

### Checklist before the final review
